src/app/src/components/ModalSections/ModalSection.js

```diff
@@ -0,0 +1,40 @@
+import {
+    Flex,
+    Heading,
+    ModalBody,
+    ModalContent,
+    ModalFooter,
+    ModalHeader,
+    Spacer,
+} from '@chakra-ui/react';
+
+export default function ModalSection({
+    preHeading,
+    heading,
+    nextButton,
+    prevButton,
+    children,
+}) {
+    return (
+        <ModalContent minH='65vh' p={8}>
+            <ModalHeader>
+                {preHeading && (
+                    <Heading size='md' color='gray.500'>
```

Comment:
Let's use a `Heading` variant to style this block:

```diff
diff --git a/src/app/src/components/ModalSections/ModalSection.js b/src/app/src/components/ModalSections/ModalSection.js
index 6265cc4..60a6d57 100644
--- a/src/app/src/components/ModalSections/ModalSection.js
+++ b/src/app/src/components/ModalSections/ModalSection.js
@@ -19,7 +19,7 @@ export default function ModalSection({
         <ModalContent minH='65vh' p={8}>
             <ModalHeader>
                 {preHeading && (
-                    <Heading size='md' color='gray.500'>
+                    <Heading as='h5' variant='preHeading'>
                         {preHeading}
                     </Heading>
                 )}
diff --git a/src/app/src/theme.js b/src/app/src/theme.js
index 5a5af18..1653108 100644
--- a/src/app/src/theme.js
+++ b/src/app/src/theme.js
@@ -39,9 +39,20 @@ const Button = {
     },
 };
 
+const Heading = {
+    variants: {
+        preHeading: {
+            color: 'gray.500',
+            fontFamily: 'body',
+            fontSize: 'md',
+        },
+    },
+};
+
 const theme = extendTheme({
     components: {
         Button,
+        Heading,
     },
     fonts: {
         heading: `'Quicksand', sans-serif`,
```

This separate the styling from the content.

Reply:
36534bf6c6c65872deb171434e8d77535ccd2ac3